app/commands/tooling_job/process.rb

```diff
@@ -40,6 +40,20 @@ def call
           JSON.parse(download_file(attrs["output"]["analysis.json"]))
         )
       end
+
+      client.update_item(
+        table_name: Exercism.config.dynamodb_tooling_jobs_table,
+        key: { id: id },
+        expression_attribute_names: {
+          "#JS": "job_status",
+          "#LU": "locked_until"
+        },
+        expression_attribute_values: {
+          ":js": "processed",
+          ":lu": nil
+        },
+        update_expression: "SET #JS = :js, #LU = :lu"
+      )
```

Comment:
The [example code you linked to](https://github.com/exercism/v3-website/issues/35#issue-674628338) used `return_values: "ALL_NEW"`. As we're not actually using the returned values from the `update_item` method, I've omitted this line for optimization purposes (see [the docs](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html)).

Reply:
Great :)